cmake/syscheck.cmake

```diff
@@ -7,7 +7,12 @@ if (WITH_ZEPHYR)
   if (NOT WITH_ZEPHYR_LIB)
     include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
   endif()
-  if (CONFIG_CPU_CORTEX_M)
-    set (MACHINE "cortexm" CACHE STRING "")
-  endif (CONFIG_CPU_CORTEX_M)
+  if (CONFIG_CPU_CORTEX_A)
+    set (MACHINE "cortex_a_m_r" CACHE STRING "")
```

Comment:
what about just "arm" ?

Reply:
yes that's better, looking for a better CONFIG_ to check, CONFIG_ARM maybe a good candidate.